Merge sidebar & New landing page design (#1011)
redpanda-data/documentation: modules/troubleshoot/pages/debug-bundle/overview.adoc

--
====

In addition to the preceding files and directories, the debug bundle also includes 

> all stack traces from the root cause of a crash loop for up to 50 crashes in FIFO order. Logs include:

Reviewer: ```suggestion
up to 50 of the most recent stack traces from the root cause of a crash loop. Logs include:
```
